habitat/core/batched_env.py

```diff
@@ -1,8 +1,6 @@
 #!/usr/bin/env python3
 
-# Copyright (c) Facebook, Inc. and its affiliates.
-# This source code is licensed under the MIT license found in the
-# LICENSE file in the root directory of this source tree.
+# Copyright (c) Meta Platforms, Inc. All Rights Reserved
```

Comment:
Are you sure about removing reference to the MIT license text? Isn't this source file still under the MIT license?

Reply:
This file was added as part of the gala_kinematic work no? I guess I can leave both of them?